scene/resources/style_box.cpp

```diff
@@ -105,6 +105,7 @@ void StyleBoxTexture::set_texture(RES p_texture) {
 	texture = p_texture;
 	emit_signal("texture_changed");
```

Comment:
I don't think so, as this signal can be nice for usage from gdscript.
_change_notify is only for the c++ side of things.